docs/structure-of-a-contract.rst

```diff
@@ -87,6 +90,47 @@ Function modifiers can be used to amend the semantics of functions in a declarat
         }
     }
 
+
+Function modifiers can override each other. This works similar as function
```

Comment:
```suggestion
Function modifiers can override each other. This works similar to function
```

Reply:
@ChrisChinchilla we need your authority in relation to the English language here. How should this be phrased?

Reply:
@erak was mostly right, but maybe one small thing.

```suggestion
Function modifiers can override each other. This works in a similar way to functions
```

Reply:
Is it similar or identical?